### DISCUSSION TITLE
Device goes 'Unavailable' when powered off and integration is hidden in HA

### DISCUSSION THREAD

**Discussion by gitothy** [ofalvai/home-assistant-candy#117] [Ideas]
Great integration, many thanks.

I've got a Hoover washing machine and whenever it is powered off or not connected the status of the related entities in HomeAssistant switch to 'Unavaliable' and automatically get hidden along with the integration.

![image](https://user-images.githubusercontent.com/39539635/169693712-c4cb851f-86c2-4d61-a14f-9cae1c7e62fa.png)

I appreciate this is a HA limitation but if the status when unreachable could be automatically set to 'Off' and the timeout before status 'Unavailable' is upped to 7 days it would give a better experience.

Thanks again.

**EDIT: Is this a dupe of https://github.com/ofalvai/home-assistant-candy/issues/12 ?**

**Comment by ofalvai** [comment]
> EDIT: Is this a dupe of https://github.com/ofalvai/home-assistant-candy/issues/12 ?

Yes, the root cause is the same I believe. There is no way to differentiate between an "off" and an "unavailable" state. Let's continue the discussion in that thread, I'm closing this issue.
